src/searching.md

```diff
@@ -3,17 +3,16 @@
 <!-- toc -->
 
 Anki's Browse screen and the Filtered Deck feature use a common method
-of searching for specific cards/notes.
+of searching for specific cards/notes. This method is also used in the search bar below "FSRS parameters" field.
```

Comment:
At a minimum this needs an article -- below ***the*** "FSRS ... 

But I don't think it's quite fitting to call that a "search bar," because that suggests it returns a result to the user. I would suggest: 
`This method can also be used to adjust the scope of FSRS optimization.`

Reply:
> Filtered decks don't give you a search result

They return a set of cards in the deck based on the filter, so I'd say they do give the user a search result. 

> do we really need to be technically correct?

Yes, that's pretty much the entire point of the manual.

Reply:
We are already using the word in the manual and in the FSRS tutorial. I added the 'bar' but I think it shouldn't be an issue.

Reply:
```suggestion
of searching for specific cards/notes.  This method can also be used to adjust the scope of FSRS optimization. 
```